src/AttributeReleasePolicy.php

```diff
@@ -11,44 +11,101 @@ class AttributeReleasePolicy
     const WILDCARD_CHARACTER = '*';
 
     /**
+     * Holds attribute rule values with optional 'source'.
+     *
+     * Non-aggregated attributes are stored in the format:
+     *
+     *   attribute name => [value, value, value]
+     *
+     * Attributes aggregated from different sources specify a source along with the value:
+     *
+     *   attribute name => [
+     *      [ 'soure' => '...', 'value' => '...' ]
+     *   ]
+     *
      * @var array
      */
-    private $attributesWithValues;
+    private $attributeRules;
 
     /**
-     * @param array $attributesWithValues
+     * @param array $attributesRules
      */
-    public function __construct(array $attributesWithValues)
```

Comment:
The parameter: `attributesWithValues` does not match the value provided in the `@param` annotation in the docblock.